pkg/registry/apis/provisioning/export.go

```diff
@@ -130,10 +129,7 @@ func (c *exportConnector) Connect(
 				continue
 			}
 
-			folder := item.GetAnnotations()["grafana.app/folder"]
-			// TODO: Use folder to find where to put it
-			slog.InfoContext(ctx, "dashboard folder found",
-				"folder", folder)
+			folder := folders[item.GetAnnotations()["grafana.app/folder"]]
```

Comment:
rather than reading the annotation directly, maybe:
```
obj, err := utils.MetaAccessor(item)
...
obj.GetFolder()
```
https://github.com/grafana/grafana/blob/main/pkg/apiserver/registry/generic/strategy.go#L57

Reply:
hymm -- ok, at least https://github.com/grafana/grafana/blob/main/pkg/apimachinery/utils/meta.go#L25